src/foam/nanos/dig/DIG.js

```diff
@@ -53,6 +53,25 @@ foam.CLASS({
     },
     'cmd',
     'format',
+{
+      name: 'digColumns',
+      expression: function(daoKey) {
```

Comment:
FYI if you make it a StringArray, you won't be able to use `expression` unless you null out the factory on it. i.e. add `factory: null` if you want the `expression` to work.